WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml.cs

```diff
@@ -17,7 +15,7 @@ public enum ReplacementMode
 
 public class PrivacyContentControl : ContentControl
 {
-	private CompositeDisposable? _disposable;
+	private static readonly TimeSpan RevealDelay = TimeSpan.FromSeconds(1.5);
```

Comment:
1.5 sec feels so long, the purpose of this would be to not reveal any information while the mouse is moving from A to B.

Maybe 0.5 or 0.75 would be sufficient.

Reply:
I've set 0.75 and I thinks it's better now. This is quite subjective, so feel free to modify to the right point :)